src/Symfony/Component/Console/Application.php

```diff
@@ -118,16 +119,40 @@ public function run(InputInterface $input = null, OutputInterface $output = null
         $this->configureIO($input, $output);
 
         try {
+            $e = null;
             $exitCode = $this->doRun($input, $output);
         } catch (\Exception $e) {
+            $exception = $e;
+        } catch (\Throwable $e) {
+            $exception = new FatalThrowableError($e);
+        }
+
+        if (null !== $e && null !== $this->dispatcher) {
+            $event = new ConsoleErrorEvent($this->runningCommand, $input, $output, $e, $e->getCode());
```

Comment:
With everything on master, is you run: `bin/console mldsqlkmldmksq` you got this:
```
Fatal error: Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Type error: Argument 1 passed to Symfony\Component\Console\Event\ConsoleErrorEvent::__construct() must be an instance of Symfony\Component\Console\Command\Command, null given, called in /home/greg/dev/github.com/lyrixx/symfony/src/Symfony/Component/Console/Application.php on line 131 in /home/greg/dev/github.com/lyrixx/symfony/src/Symfony/Component/Console/Event/ConsoleErrorEvent.php:30
Stack trace:
#0 /home/greg/dev/github.com/lyrixx/symfony/src/Symfony/Component/Console/Application.php(131): Symfony\Component\Console\Event\ConsoleErrorEvent->__construct(NULL, Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput), Object(Symfony\Component\Console\Exception\CommandNotFoundException), 0)
#1 /home/greg/dev/github.com/lyrixx/symfony-standard/bin/console(28): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput))
#2 {main}
  thrown in /home/greg/dev/github.com/lyrixx/symfony/src/Symfony/Component/Console/Event/ConsoleErrorEvent.php on line 30
```

It's weird nobody notice that. May be I'm missing something but I really doubt, because all my repository are clean.

1. Do you experience this behaviour too?
2. How to patch that ? I can do a PR, but I'm not sure about the how to solve this.